HW03/ihar_shkuryn/ihar_shkuryn_hw_03_t_01.rb

```diff
@@ -1,7 +1,10 @@
 # frozen_string_literal: true
 
+# get log as a string and return line including keyword 'error'
 class Homework3
   def task1(log)
+    return '' unless log.respond_to? :split
```

Comment:
as far as I remember, split method is only for strings. so I would rather check for log being an instance of String class